### DISCUSSION TITLE
YouTube not working

### DISCUSSION THREAD

**Discussion by HagarJNode** [lavalink-devs/lavaplayer#142] [Q&A] [answered]
Am I the only one who have issues with the YouTubeTracker not wanting playing? I tryed the new version 2.2.1 and the latest snapshot version, but no luck.

**Comment by topi314** [comment, marked as answer]
please read the changelog
https://github.com/lavalink-devs/lavaplayer/blob/main/CHANGELOG.md#212---2024-05-05

**Reply by HagarJNode** [reply]
Thanks a lot, upgrading worked!
